ci/list.sh

```diff
@@ -26,16 +26,19 @@ then
         fi
     done
 else
+
     if [ $TRAVIS_TAG ]
     then
         stack_id=`echo ${TRAVIS_TAG/-v[0-9]*/}`
         echo "Listing stacks for this release"
+    elif [ -z $STACKS_LIST ]
+    then
+        echo "You need to set a value for the following environment variables:"
```

Comment:
I do not think we need this. `list.sh` will set this variable if it is not set

Reply:
Thank you, I have fixed this in the most recent commit